xlive/Blam/Engine/main/interpolator.h

```diff
@@ -71,33 +85,32 @@ void halo_interpolator_update_end(void);
 bool halo_interpolator_interpolate_center_of_mass(datum object_datum, real_point3d* center_of_mass);
 
 bool halo_interpolator_interpolate_object_node_matrices(datum object_index, real_matrix4x3** node_matrices, int32* out_node_count);
-bool halo_interpolator_interpolate_weapon_node(datum user_index, datum animation_index, int32 node_index, int32 weapon_slot, real_matrix4x3* out_node);
 
 void halo_interpolator_setup_new_object(datum object_datum);
 
-void halo_interpolator_setup_weapon_data(int32 user_index, datum animation_index, int32 weapon_slot, real_matrix4x3* node_matrices, int32 nodes_count);
+void halo_interpolator_setup_weapon_data(uint32 user_index, datum animation_index, int32 weapon_slot, const real_matrix4x3* node_matrices, uint32 nodes_count);
```

Comment:
why did you change the type of user_index?

Reply:
NONE never ends up being passed in here but I can change it back